react/features/conference/components/index.js

```diff
@@ -1 +1,4 @@
+// @flow
+
 export { default as Conference } from './Conference';
+export { default as Notice } from './Notice';
```

Comment:
I don't think we want to export this, since it's just used internally by Conference.

Reply:
`Notice` is exported with the `export default expression;` syntax ( in `export default translate(connect(_mapStateToProps)(Notice));`).

To be imported with the `Notice` name, the only thing that works for me at this point is to export it  with the `as` syntax (`export { default as Notice } from './Notice';`). Unfortunately, the only way to do this is using `export { import1 as name1, import2 as name2, …, nameN } from …;` 

Possible is doable in some how but I don't see any way to do it at this point. Maybe a example could be useful for me.

Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export

Reply:
Answering myself ...

```
diff --git a/react/features/conference/components/Conference.web.js b/react/features/conference/components/Conference.web.js
index bf433f9d..5a3cc838 100644
--- a/react/features/conference/components/Conference.web.js
+++ b/react/features/conference/components/Conference.web.js
@@ -13,7 +13,7 @@ import { CalleeInfoContainer } from '../../invite';
 import { LargeVideo } from '../../large-video';
 import { NotificationsContainer } from '../../notifications';
 import { SidePanel } from '../../side-panel';
-import { Notice } from './';
+import { default as Notice } from './Notice';
 import {
     Toolbox,
     fullScreenChanged,
diff --git a/react/features/conference/components/index.js b/react/features/conference/components/index.js
index 92bdad30..0910dcaf 100644
--- a/react/features/conference/components/index.js
+++ b/react/features/conference/components/index.js
@@ -1,4 +1,3 @@
 // @flow
 
 export { default as Conference } from './Conference';
-export { default as Notice } from './Notice';